.github/workflows/workflow-validate-repo-markdown.yml

```
@@ -31,42 +26,33 @@ jobs:
+    - name: Create issue if bad links detected
+      if: ${{ !cancelled() && env.lychee_exit_code != 0 && inputs.linkcheck_create_issue }}
+      uses: peter-evans/create-issue-from-file@e8ef132d6df98ed982188e460ebb3b5d4ef3a9cd
       with:
         title: Scheduled Check of Markdown Documents Found Bad Hyperlinks
-        content-filepath: mlc_report.log
+        content-filepath: markdown-link-report.md
         labels: |
           bug
           documentation
+    - name: Fail on link check error
+      if: ${{ !cancelled() && env.lychee_exit_code != 0 && (github.event.inputs.linkcheck_fail_on_error == 'true' || inputs.linkcheck_fail_on_error == true) }}
```

actionlint (1.7.4)

property "linkcheck_fail_on_error" is not defined in object type {ignorepattern: string; linkcheck_create_issue: bool}

(expression)